### ISSUE TITLE
[BUG] Hash mismatch on mainnet-block 10542500 with v0.5.22 client 

### ISSUE THREAD

**Issue by jun0tpyrc** [terra-money/classic-core#801]
**Describe the bug**
[BUG] Hash mismatch on mainnet-block 10542500 with v0.5.22 client 
```
2:17PM INF Replay last block using real app module=consensus
Error: error during handshake: error on replay: wrong Block.Header.AppHash.  Expected 19BF853E8360CDCD8951CC8AFA5D0E26C0B2B587F9C422E2C7C7AD6DA579D013, got E813C1A1BBC0A92BF25DA6104FC0754349B42FC1B608B36592CE1D7B58B8B10F
Usage:
```

**Comment by gitmewai** [comment]
Will the prebuilt binaries also be released under `v0.5.23`? Thanks

**Comment by gregnuj** [comment]
v0.5.23 release and binary available binary is equivalent to terra-rebels referenced above
https://github.com/terra-money/classic-core/releases/tag/v0.5.23
